src/authz-module/libraries-manager/components/AddNewTeamMemberModal/messages.ts

```diff
@@ -53,22 +53,22 @@ const messages = defineMessages({
   },
   'libraries.authz.manage.add.member.success': {
     id: 'libraries.authz.manage.add.member.success',
-    defaultMessage: '{count, plural, one {# team member added successfully.} other {# team members added successfully.}}',
+    defaultMessage: '{count, plural, one {# team member added successfully} other {# team members added successfully}} ({userIds})',
     description: 'Success message when adding new team members',
   },
-  'libraries.authz.manage.add.member.failure': {
-    id: 'libraries.authz.manage.add.member.failure',
-    defaultMessage: '<Bold>We couldn\'t find a user for {count, plural, one {# email address or username.} other {# email addresses or usernames.}}</Bold><Br></Br> Please check the values and try again, or invite them to join your organization first.',
-    description: 'Error message when adding new team members',
+  'libraries.authz.manage.add.member.failure.not.found': {
+    id: 'libraries.authz.manage.add.member.failure.not.found',
+    defaultMessage: '<Bold>We couldn\'t find a user for {count, plural, one {# email address or username} other {# email addresses or usernames}} ({userIds}).</Bold><Br></Br> Please check the values and try again, or invite them to join your organization first.',
```

Comment:
This is a comment not blocking, but I think the user identifier should be part of the body of the notification, like: 

```suggestion
    defaultMessage: '<Bold>We couldn\'t find a user for {count, plural, one {# email address or username} other {# email addresses or usernames}}.</Bold><Br></Br> Please check the values ({userIds}) and try again, or invite them to join your organization first.',
```